beacon_node/store/src/impls/beacon_state.rs

```diff
@@ -4,24 +4,20 @@ use ssz_derive::{Decode, Encode};
 use std::convert::TryInto;
 use types::beacon_state::{CloneConfig, CommitteeCache, CACHED_EPOCHS};
 
-pub fn store_full_state<KV: KeyValueStore<E>, E: EthSpec>(
-    store: &KV,
+pub fn store_full_state<E: EthSpec>(
     state_root: &Hash256,
     state: &BeaconState<E>,
+    ops: &mut Vec<KeyValueStoreOp>,
 ) -> Result<(), Error> {
-    let total_timer = metrics::start_timer(&metrics::BEACON_STATE_WRITE_TIMES);
```

Comment:
It would be meaningless.  The PR makes it so that the actual db write is executed at some other place in the code, during the call to `do_atomically()`.

Reply:
Oh, of course! Lets delete `BEACON_STATE_WRITE_TIMES` from [`beacon_node/store/src/metrics.rs`](https://github.com/sigp/lighthouse/blob/master/beacon_node/store/src/metrics.rs) then, as it's now unused. I've also opened #1307 to track the state of the store metrics

Reply:
Done ✅